MobilityData/mobility-feed-api
# chore: Hide new home and add feeds screens under feature tags

**Summary:**

Closes #586 

- Added _enableFeedsPage_ feature flag.
  - true: renders the home page with search feed capabilities
  - false:  renders _previous/legacy_ static home page
- Added _enableFeedSubmissionStepper_ feature flag.  
  -  true: renders the feed submission stepper based on the FeedSubmissionStepper.tsx 
  - false: renders the legacy feed submission page based on the Contribute.tsx
- Removed _enableMVPSearch_ that had no connected functionality. 

**Testing tips:**

 - The _enableFeedsPage_ shows the search features
   - Go to firebase console(dev environment)
   - Make sure the _enableFeedsPage_ is set to true. If it is set to false, set and publish the remote config to true and wait ~5 minutes to apply.
   - Visit the PR deployed app link.
   - Expected the home page to show the search text and icon.

 - The _enableFeedsPage_ shows the legacy screen
   - Go to firebase console(dev environment)
   - Make sure the _enableFeedsPage_ is set to false. If it is set to true, set and publish the remote config to true and wait ~5 minutes to apply.
   - Visit the PR deployed app link.
   - Expected the home page to show the current/legacy home screen.
   
 - The _enableFeedSubmissionStepper_ shows the new add feed screen
    - Go to firebase console(dev environment)
   - Make sure the enableFeedSubmissionStepper_ is set to true. If it is set to false, set and publish the remote config to true and wait ~5 minutes to apply.
   - Visit the PR deployed app link.
   - Expected the add feed page to show the new(not completed) HTML assistant.
   
 - The _enableFeedSubmissionStepper_ shows the current Google form screen
    - Go to firebase console(dev environment)
   - Make sure the enableFeedSubmissionStepper_ is set to false. If it is set to true, set and publish the remote config to true and wait ~5 minutes to apply.
   - Visit the PR deployed app link.
   - Expected the add feed page to show the current Google form.
  
Please make sure these boxes are checked before submitting your pull request - thanks!
